.circleci/config.yml

```diff
@@ -21,6 +21,10 @@ shared: &shared
     - POSTGRES_PASSWORD=nodepw
     - POSTGRES_DB=nodedb
     - MEMCACHED=localhost:11211
+    - MSSQL_HOST=localhost
+    - MSSQL_PORT=1433
+    - MSSQL_USER=sa
+    - MSSQL_PW=stanCanHazMsSQL1
```

Comment:
These were missing at all in the circle ci config :)

Reply:
Probably not required because the default values work for CircleCI, but +1 to add them here as well for consistency. 👍

Reply:
FYI Tests failed in CircleCi because of the missing envs :)

Reply:
Ah, right! Now that we set them in the code if they are not present, they are absolutely required here. Didn't think that through properly 👍